sensor_msgs/Image image
string type
float32 gps_lati
float32 gps_longit
string target_color
string target_shape
string symbol
string symbol_color
uint32 orientation
string description
